sql/301_Grafana_Mapping_Create_table.sql

```diff
@@ -0,0 +1,11 @@
+--- Create Grafana Dashboard Mapping Table ---
+drop table if exists public.grafana_dashboard_mapping;
+create table public.grafana_dashboard_mapping
+(
+  org_id int not null,
```

Comment:
ich sehe da keinen bedarf für. die grafana server würden die gleiche Datenbank nutzen, somit auch immer die gleiche org_id haben. das sollte so out of the box klappen. und ansonsten: das ist eh out of scope. wir haben nur ein grafana.